blackshadev/anderwijs_website
# Add upcoming events to agenda

Context, the Wordpress JSON importer and gatsby aren't the best of friends, it seems to lagg behind in updates while the rest update perfectly fine. We also do not need the JSON importer since gatsby can retrieve the data from aas directly.


## How to test
- Go to the "kampagenda" 
- See no noticeable difference with the production page Ignoring the following
  - Apart from having more events
  - And the text surrounding the table (I will need to create a replacer for a singular blok and change the production page for that)
 
## After merge
- Change the replacer in this MR to only replace one blok instead of the whole "kampagenda" page
- Change the "kampagenda" page to include said block, removing the JSON Importer blok
